src/components/Rating.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+
+function Rating(props) {
+    const {children} = props
+    function stars (children) {
+        if(children >= 4.5) return "★★★★★";
```

Comment:
A better and "easier" way to do this would be:

```
function Rating(props) {
    const {children} = props

    const rounded = Math.round(children)
  return (
    <div>
        {'★'.repeat(rounded) + "☆".repeat(5 - rounded)}; 
    </div>
  )
}
```